README.md

```diff
@@ -41,28 +41,26 @@ $ sudo gem install bourbon
 $ sudo gem install neat
 
 # Then, navigate to the directory to install Bourbon into
-$ cd /scaffold/libraries
+$ cd path/to/scaffold/libraries
 $ bourbon install
 $ neat install
 ```
 
 ### Compile manually
-While working on the Scaffold, compile your CSS and test the output by running it in this folder. Here's the command: 
+While working on the Scaffold, compile your CSS and test the output by running it in this folder. Here's the command:
 
 ```sh
-$ sass --watch styles.scss:output.css --style compressed
+$ sass --watch path/to/styles.scss:path/to/output.css --style compressed
 ```
 
-### Run Lint
-We use the Brigade Linter gem: <https://github.com/brigade/scss-lint>
+### Linting
+Linting on the scaffold uses [pre-commit by Yelp](http://pre-commit.com/) software.
 
-Install: `gem install scss_lint`
+##### Installing Pre-Commit
+If you Homebrew, `brew install pre-commit`
+If not, freakin install [Homebrew](http://brew.sh/)
 
-Run `scss-lint` from the command line by passing in a directory (or multiple directories) to recursively scan:
-
-```sh
-# from the root of your system, set the path to this Scaffold
-$ scss-lint ~/path/to/scaffold/ 
-```
+Currently the linter is setup to only run on changed files and only when commiting,
+To override the linter you can use the `--no-verify` flag when doing your commit.
```

Comment:
Might be useful to give an example of this.. especially how it can be used in the middle of a code block for excluding a couple lines of code.